test-harness/specs/security-scheme-validation-AC-2_failing.oas2.txt

```diff
@@ -0,0 +1,28 @@
+====test====
+Making a request (and not including any credentials) to an endpoint,
+which security depends on the one and only global security definition,
+results in a 401 status code response.
+====spec====
+swagger: "2.0"
+paths:
+  /todos:
+    get:
+      responses:
+        200:
+          description: OK
+          schema:
+            type: string
+            example: test
+securityDefinitions:
+  BasicAuth:
+    type: http
+    scheme: basic
+security:
+  - BasicAuth: []
+====server====
+mock -p 4010
+====command====
+curl -sIXGET http://localhost:4010/todos
+====expect====
+HTTP/1.1 401 Unauthorized
+WWW-Authenticate: Basic realm="todos"
```

Comment:
I'm also not sure about `realm` thing. Maybe we could just always send back `realm="*"` for such cases? Here I just took the name of the operation

Reply:
Let's do `"*"` or `"prism"`, this parameter is arbitrary and depends on how server implements it. If we ever support a plugin system for Prism this is a good candidate for a plugin.